openpype/hosts/blender/api/lib.py

```diff
@@ -315,3 +333,184 @@ def maintained_time():
         yield
     finally:
         bpy.context.scene.frame_current = current_time
+
+
+def download_last_workfile(
+    project_name: str, asset_name: str, task_name: str
+) -> str:
+    """Download last workfile and return its path.
+
+    Args:
+        project_name (str): Name of project.
+        asset_name (str): Name of asset.
+        task_name (str): Name of task.
+
+    Returns:
+        str: Path to last workfile.
+    """
+    from openpype.modules.sync_server.sync_server import (
+        download_last_published_workfile,
+    )
+
+    sync_server = ModulesManager().get("sync_server")
+    if not sync_server or not sync_server.enabled:
+        raise RuntimeError("Sync server module is not enabled or available")
+
+    anatomy = Anatomy(project_name)
+    asset_doc = get_asset_by_name(
+        project_name,
+        asset_name,
+    )
+    family = "workfile"
+
+    filtered_subsets = [
+        subset
+        for subset in get_subsets(
+            project_name,
+            asset_ids=[asset_doc["_id"]],
+            fields=["_id", "name", "data.family", "data.families"],
+        )
+        if (
+            subset["data"].get("family") == family
+            # Legacy compatibility
+            or family in subset["data"].get("families", {})
+        )
+    ]
+    if not filtered_subsets:
+        raise RuntimeError(
+            "Not any subset for asset '{}' with id '{}'".format(
+                asset_doc["name"], asset_doc["_id"]
+            )
+        )
+
+    # Match subset wich has `task_name` in its name
+    low_task_name = task_name.lower()
```

Comment:
Why not use it directly like `task_name.lower()` in line 390?

Reply:
Why did I do this in the first place <.<